typebinder_web_playground/index.html

```diff
@@ -5,17 +5,23 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/assets/bulmaswatch-default.min.css">
```

Comment:
The folder `assets` is `.gitignore`-ed, this file is not found when launching locally. Please add it if the license allows it.

Reply:
It's MIT.
Done.